### DISCUSSION TITLE
Getting HttpClient.Timeout Issue Using Get-pnpflow command

### DISCUSSION THREAD

**Discussion by amardhanju** [pnp/powershell#1871] [Q&A]
Get-pnpflow -Environment $environment -Connection $prjConn -AsAdmin
The request was canceled due to the configured HttpClient.Timeout of 100 seconds elapsing.
Same error while calling Get-PnPFlowRun

**Comment by dougieG123** [comment]
We are experiencing the same issue. did you find a resolution to this?

**Reply by Adhanj** [reply]
Nope. I gave up :-)

**Comment by Offbeatmammal** [comment]
getting the same intermittently on a number of PnP functions. Would be really nice to have an option to set a different timeout.
Bug https://github.com/pnp/powershell/issues/4899 seems to be re this, so fingers crossed.
